TODO.md

```diff
@@ -68,3 +68,5 @@
   every read/write range command.
 
 * Gossip should be changed to send only protobufs.
+* persist Gossip and the kv.leaderCache to make sure that restarting
+  a node is harmless.
```

Comment:
Definitely worth persisting gossip. Rolling restarts will otherwise generate a lot of unnecessary network traffic. Not a big hurry to get this done, but will become a must have with larger clusters.